https://github.com/RadeonOpenCompute/ROCT-Thunk-Interface/pull/42

From 9090559aad755bf2b5041fb617c3ddf790c7cdf6 Mon Sep 17 00:00:00 2001
From: Craig Andrews <candrews@integralblue.com>
Date: Thu, 8 Aug 2019 14:01:04 -0400
Subject: [PATCH] hsakmt-dev should not install linux kernel headers

hsakmt-dev should not install include/linux/* (currently just kfd_ioctl.h) as those are linux kernel headers provided by the linux kernel header packages (`linux-headers-*` on Debian/Ubuntu or `kernel-headers-*` on Red Hat / Fedora)
---
 hsakmt-dev.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hsakmt-dev.txt b/hsakmt-dev.txt
index eb0d3e2..6af364c 100644
--- a/hsakmt-dev.txt
+++ b/hsakmt-dev.txt
@@ -44,7 +44,7 @@ set ( CMAKE_VERBOSE_MAKEFILE on )
 
 ## Set the install targets
 install ( FILES libhsakmt.pc DESTINATION ${CMAKE_INSTALL_PREFIX}/share/pkgconfig )
-install ( DIRECTORY ${SOURCE_DIR}/include/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/${HSAKMT_COMPONENT} )
+install ( DIRECTORY ${SOURCE_DIR}/include/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/${HSAKMT_COMPONENT} PATTERN "linux" EXCLUDE )
 
 
 ## Set the default generator types for the devel package.
